big-red-shuttle/Network.swift

```diff
@@ -23,17 +27,17 @@ class System {
     }
     
     func userLocation() -> Coordinate? {
-       return Location.sharedLocation.currentUserLocation
+       return Location.shared.currentUserLocation
     }
 
     func busLocation() -> Coordinate? {
-        return Location.sharedLocation.currentBusLocation
+        return Location.shared.currentBusLocation
```

Comment:
probs don't need these, GPS and Location can be used to get current bus location